canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -181,7 +181,7 @@ $palette-border-color: $ui-03;
 .palette-flyout-category {
 	cursor: pointer;
 	border-width: 0 0 0 $palette-border-width;
-	height: 46px;
+	height: 60px;
```

Comment:
I mentioned in slack that we should keep the height the same - as long as two lines of text display OK it will be alright.  
To display text greater than 2 lines you can set this to:
```
height: fit-content;
min-height: 46px; 
```
and remove the `height` attribute from the `palette-flyout-category-item` styles.